samples/subsys/logging/logger/src/main.c

```diff
@@ -29,6 +28,10 @@ SAMPLE_INSTANCE_DEFINE(inst1);
 #define INST2_NAME STRINGIFY(SAMPLE_INSTANCE_NAME.inst2)
 SAMPLE_INSTANCE_DEFINE(inst2);
 
+#ifndef NRF_RTC1
```

Comment:
`#ifdef CONFIG_SOC_FAMILY_NRF` would be better

Reply:
For #9988 it wouldn't be.